src/v8runner/src/v8runner.os

```diff
@@ -149,6 +149,18 @@
 
 КонецПроцедуры
 
+Процедура СнятьКонфигурациюСПоддержки(Знач Принудительно = Ложь) Экспорт
+
+	Параметры = СтандартныеПараметрыЗапускаКонфигуратора();
+	Параметры.Добавить("/ManageCfgSupport -disableSupport");
+	Если Принудительно Тогда
+		Параметры.Добавить("-force");
```

Comment:
Если в настройках поддержки не включена возможность изменения, то без
"принудительно" кританет.

20 июля 2016 г. 9:51 AM пользователь "Andrei Ovsiankin" <
notifications@github.com> написал:

> In src/v8runner/src/v8runner.os
> <https://github.com/EvilBeaver/oscript-library/pull/131#discussion_r71473576>
> :
>
> > @@ -149,6 +149,18 @@
> >
> >  КонецПроцедуры
> >
> > +Процедура СнятьКонфигурациюСПоддержки(Знач Принудительно = Ложь) Экспорт
> > +
> > +	Параметры = СтандартныеПараметрыЗапускаКонфигуратора();
> > +	Параметры.Добавить("/ManageCfgSupport -disableSupport");
> > +	Если Принудительно Тогда
> > +		Параметры.Добавить("-force");
>
> Кстати, а что означает "Принудительно"? В каких случаях нужно?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/EvilBeaver/oscript-library/pull/131/files/2140d5121e56bf2e7ac3987dee4d9cb11ba799cd#r71473576>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ABFJKMMyLgHOuNYOrmQ0PNFSpvVCQS4jks5qXcVjgaJpZM4JQDQl>
> .
>